.changes/next-release/bugfix-a462f2ce-093b-44ad-bf57-d506e715121b.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix double click opening Cloud Formation stack viewer"
```

Comment:
This sounds like we the opposite of what we did.

Fix double click not*

or 

Enable double click*

Reply:
Makes sense, reworded